chess/src/Components/Header.js

```diff
@@ -11,18 +11,15 @@ class Header extends React.Component {
   }
 
   render() {
-    if (!this.props.userData.user) return <h2>Logo</h2>;
+    // if (!this.props.userData.user) return <h2>Logo</h2>;
```

Comment:
Edit comment so header is conditionally rendered depending on whether a user is logged in